client/src/main/java/tds/assessment/Form.java

```diff
@@ -125,4 +137,27 @@ public Long getUpdateVersion() {
         return updateVersion;
     }
 
+    /**
+     * @return The {@link tds.assessment.Item}s associated with this form
+     */
+    public List<Item> getItems() {
+        return items;
+    }
+
+    public void setItems(List<Item> items) {
+        this.items = items;
+    }
+
+    /**
+     * Determine the number of {@link tds.assessment.Item}s associated with this form.
+     *
+     * @return The number of {@link tds.assessment.Item}s in this form.  If the list is null, return 0.
+     */
+    public int getLength() {
+        if (items == null) {
+            items = new ArrayList<>();
```

Comment:
I'm not sure I see a benefit to setting this here instead of just returning 0.  `return item == null ? 0 : items.size()`